### ISSUE TITLE
Design exploration: Improve discoverability of Vet Center Satellite Locations

### ISSUE THREAD

**Issue by thejordanwood** [department-of-veterans-affairs/va.gov-cms#18121]
## Description 

For [Vet Centers 2.0](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17661), explore how Vet Center satellite locations could be discoverable in their own right. Notes in a previous ticket state, "Because they appear on the Locations page of the parent Vet Center, they aren't findable in their own right. Veterans do not know to search for these locations in the context of a main Vet Center." (Ticket #17693)

For more context: In [the research synthesis](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/vet-centers/initiatives/2024-vet-center-2.0/seconday-research-synthesis/Expand%20satellite%20locations.md), past research indicates that Veterans understand “Satellite locations” as related to the main location and that they are often smaller.

## Design 
[Vet Center Figma file](https://www.figma.com/file/EVd3q06ukAbVS61Q8MwRAT/Vet-Centers?type=design&node-id=1%3A900&mode=design&t=Wqtm0fJVxdIkoObB-1)

## Design considerations
- Consider how to support expansion of satellite locations. 
     - Only Vet Centers have pages and associated services. Other locations where services are available are included on the Locations page for the parent Vet Center. Some satellite locations are very similar in maturity to a full Vet Center and may have dedicated staff, daily hours and a full suite of services. Without their own page, they can be difficult to discover because Veterans do not know to search for these locations in the context of a main Vet Center.
     - Vet Center editors do not have a way to indicate the services available or details about the ways the service is delivered, such as evidence based modalities or support groups for either of these types of locations. In some cases, this could prevent satellite locations from showing on Facility Locator search results.
- Consider making the link to the locations page more prominent. The link to "View more [name] Vet Center locations" is currently below the Location and contact information and is displayed in normal text which may blend in with surrounding content.
- Consider adding a side nav to support IA and navigation to other locations. Related ticket #17195 

## Acceptance criteria

- [x] Design mockups exploring different options are documented in Figma
- [ ] Review with UX Lead, Product, Engineering, Content, and Accessibility
- [ ] Design is reviewed with and sent to VA UX Lead for final sign off
- [ ] Follow up tickets are created

**Comment by thejordanwood** [comment]
I started a design for this in the [Vet Centers Figma file](https://www.figma.com/design/EVd3q06ukAbVS61Q8MwRAT/Vet-Centers?node-id=21%3A2&t=7GpQalrIlIAa9cwA-1) and am going to work on getting reviews.

**Comment by davidmpickett** [comment]
notes from UX sync:
- "How are we different" box should move below "On this page"
-  Need to choose whether we are using action links or active links for "See all locations"
- Design removes some headers, puts phone number in one line
- It removes some language that might be more self-evident
- CAIA has guidance to make links distinct from headings, rather than header links. We're going to try to design with this in mind.
- Michelle has a meeting relating to which kinds of satellite locations should get their own pages.

**Comment by thejordanwood** [comment]
Here's my pre-OOO update. I've been doing this work in the [Vet Centers Figma file](https://www.figma.com/design/EVd3q06ukAbVS61Q8MwRAT/Vet-Centers?node-id=21%3A2&t=7GpQalrIlIAa9cwA-1).

**What I've done since UX sync:**
- "How are we different" box was moved below "On this page"
- I've chosen active links for "See all locations"
- I've shown an example of what it would look like if we made links distinct from headings on the location pages

**Next steps for this:**
- Reviews!
     - @davidmpickett and @laflannery have reviewed and added comments. Please add more if anything else comes to mind
     - @eselkin and @omahane Can you review this? I think the main change will be adding a side nav to these pages. Dave pointed out some of the smaller changes in the comment above. 
     - @aklausmeier I've tried to prepare the Figma doc for your review in my absence. There's comments calling out changes and asking questions. We'll be able to sync up on this when I return on 6/6.

**Comment by jilladams** [comment]
Adding @omahane and @eselkin as assignees for awareness about eng review needed during Sprint 5.

**Comment by mmiddaugh** [comment]
@aklausmeier @thejordanwood Please include consideration for how locations are sorted and displayed. We're getting reports that the list for MVCs may have regressed recently. The sort order should prioritize those locations which are actually affiliated with the parent Vet Center, followed by an alphabetized list of those which were added as "additional" or "nearby". 

Reference: [#18229 Sort order for Vet Center Satellite locations](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/18229)

**Comment by aklausmeier** [comment]
reviewed design and left comments in Figma for Jordan

**Comment by mmiddaugh** [comment]
See also: [Map thumbnails for Vet Center CAPs do not render on the locations page #15656](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/15656)

**Comment by aklausmeier** [comment]
@thejordanwood documented accessibility improvements from Laura's [previous Vet Center audit](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17371#issuecomment-2043127994) in the Figma file, don't want to lose sight of these insights

**Comment by thejordanwood** [comment]
Update on the work that's been done to this recently. I thought this might be helpful for those going to the onsite next week, since you'll have limited availability for reviews at that time. @mmiddaugh @aklausmeier  @davidmpickett

**Feedback I've implemented:**
- On the outstation location page - I added a "Other nearby Vet Centers" section to the bottom of the page that links to the locations page. 
- On the locations page - I added a map thumbnail to the CAP example. 
- On the locations page - I've tried to better document how locations are sorted and displayed. Dave had some additional information on how the sorting works in [this Figma comment](https://www.figma.com/design/EVd3q06ukAbVS61Q8MwRAT?node-id=21-2#826332038). I've added a fake Nashville Mobile Vet Center to show that mobile vet centers are sorted in this order:
     - Mobile Vet Center affiliated with the Main location
     - All other Mobile Vet Centers by alpha

**Next steps for this:**
- I'm looking into the summary box and am brainstorming other solutions that will allow us to keep it above the "On this page" component.

**Comment by eselkin** [comment]
For satellite locations, I guess my questions are for @omahane 

Will CMS make it possible to query things like `prepare_for_your_visit` and `field_health_services` from the satellite facility (like some reverse query on a main location or maybe something less complex)? Or are satellites getting their own values/connections for these fields? Provided we can get that data on a query about the satellite facilities, these things seem ok and the design seems good.

**Comment by thejordanwood** [comment]
End of sprint update: 

- This will need to roll into next sprint to give everyone time to review. Engineers will need to review this and everyone that had initial feedback will want to review what I've implemented ([listed in this comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18121#issuecomment-2155162431)).
- I explored other solutions for the summary box and left some comments in the Figma file about it. This portion won't hold up the ticket because we can continue discussing this as the design progresses in the [next design ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15349). 

cc: @jilladams @Agile6MSkinner

**Comment by omahane** [comment]
TL;DR: I don't see any red flags.

I looked at this yesterday. I don't see anything that breaks what we do in Drupal or will require fundamental restructuring of the code (as far as I can tell). It follows patterns we already have in place and we've even done some code to identify the facility types that don't get their own pages yet, meaning that removing them from that group can be accomplished relatively easily from the Drupal side. 

I have the feeling that the menuing and breadcrumbs may be some work, or at least I want to spike on that before taking up that work.

**Comment by davidmpickett** [comment]
> For satellite locations, I guess my questions are for @omahane
> 
> Will CMS make it possible to query things like `prepare_for_your_visit` and `field_health_services` from the satellite facility (like some reverse query on a main location or maybe something less complex)? Or are satellites getting their own values/connections for these fields? Provided we can get that data on a query about the satellite facilities, these things seem ok and the design seems good.

They will likely be getting their own values for those fields. I haven't done the content model yet, but that is the most logical path. Alternatively, If we want to surface content from the main location on the satellite pages, I'll account for that in #17695.

**Comment by jilladams** [comment]
Closing per scrum discussions.

**Comment by Agile6MSkinner** [comment]
Edited link to figma designs to point directly to the correct element.
